### PR TITLE
Avoid unnecessary transceiver creation by using no-video policy for content share

### PR DESCRIPTION
**Pull request by hensmi-amazon**
**Issue #:** N/A

**Description of changes:** 
TLDR: It probably could be reproduced without content but basically,
* The content share peer connection actually subscribes to all remote videos but the backend just marks all sections as inactive (has always been like this).
* On the backend, we label the inactive sections with the static label inactive, as some multiple year old mitigation I added for a Firefox Unsignaled SSRC bug.
* [libwebrtc now rejects the SDP](https://webrtc.googlesource.com/src/+/6a8236617dd9416ea0fe7767166cc8e8c729d5e6) if there are two sections with the same msid like that, since the spec.

I'm pushing a fix on the backend to disambiguate the dummy MSIDs in the next week which fully mitigates this issue, but felt like removing the unnecessary subscriptions from the client side as well.

**Testing:**
Mitigates issue without aforementioned backend fix.

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
1. Join 2 attendees with video
2. Enable content share

For unrelated demo fix
1. Join replica meeting and promote
2. Enable video
3. Demote
4. Demo should automatically turn off video

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
y

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
n

5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
n

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: demos/browser/app/meetingV2/meetingV2.ts [hunk 1]
```diff
@@ -1586,9 +1586,10 @@ export class DemoMeetingApp
       this.toggleButton(button, isPromoted ? 'off' : 'disabled');
     }
 
-    // Additionally mute audio so it's not in an unexpected state when demoted
+    // Additionally mute audio and stop local video so it's not in an unexpected state when demoted
     if (!isPromoted) {
       this.audioVideo.realtimeMuteLocalAudio();
+      this.audioVideo.stopLocalVideoTile();
```
Comment (hensmi-amazon) [inline demos/browser/app/meetingV2/meetingV2.ts:1592]: unrelated demo fix :)

Comment (ltrung) [reply demos/browser/app/meetingV2/meetingV2.ts:1592]: Can we add a test case for this in the PR description :D

File: src/contentsharecontroller/DefaultContentShareController.ts [hunk 2]
```diff
@@ -39,6 +40,7 @@ export default class DefaultContentShareController
     contentShareConfiguration.credentials.joinToken =
       configuration.credentials.joinToken + ContentShareConstants.Modality;
     contentShareConfiguration.meetingFeatures = configuration.meetingFeatures.clone();
+    contentShareConfiguration.videoDownlinkBandwidthPolicy = new NoVideoDownlinkBandwidthPolicy();
```
Comment (ltrung) [inline src/contentsharecontroller/DefaultContentShareController.ts:43]: My bad. We missed this since the aeon of content share :))

**Review by ltrung — APPROVED** [review]

File: demos/browser/app/meetingV2/meetingV2.ts (hunk 1 above)
Comment (hensmi-amazon) [reply demos/browser/app/meetingV2/meetingV2.ts:1592]: too much pain to get integration tests to pass nowadays, I just let these changes sit in my local until i have a more critical PR :P
